### ISSUE TITLE
Annotations not working on multiple series with multiple y axes 

### ISSUE THREAD

**Issue by YousraLED** [blacklabel/annotations#44]
Hi, I am trying to use the annotations on my data that has multiple y axes that have very different mins and maxes. I am trying to link the annotations to different series by using the series ID however the annotation only appears on the first series (I think it does that by default or something). If I use the the second series's ID. the annotation appears as if its using the first series's axis.  
I tried adding annotations on data with multiple series but only one y-axis and the annotations appeared properly. Is this a bug or is am I missing something? 

Thanks

**Comment by pawelfus** [comment]
Indeed, it looks like something is hardcoded. Have you tried to set `annotation.yAxis`? If this not resolve the issue, then could you setup jsfiddle demo with that issue? We will check this.

**Comment by sebastianbochan** [comment]
The annotation is added only for the first yAxis.

Demo:
- http://jsfiddle.net/dnwf9fs3/

Try to add any shape to second pane. Not possible.

**Comment by morganfree** [comment]
Demo with new annotations with multiple y-axes - http://jsfiddle.net/7zomjrc0/1/
